extensions/blocks/contact-form/components/jetpack-crm-connection-settings.js

```diff
@@ -114,15 +110,16 @@ function CRMConnectionSettings( props ) {
 	};
 
 	return (
-		<BaseControl>
+		<PanelRow>
 			{ isFetchingPlugins && <Spinner /> }
 
 			{ shouldDisplayToggle() && (
 				<ToggleControl
-					label={ __( 'CRM Connection', 'jetpack' ) }
+					className="jetpack-contact-form__crm_toggle"
+					label={ __( 'Jetpack CRM', 'jetpack' ) }
 					checked={ jetpackCRM }
 					onChange={ value => setAttributes( { jetpackCRM: value } ) }
-					help={ __( 'Enable and disable Jetpack CRM integration for this form.', 'jetpack' ) }
+					help={ __( 'Store in CRM [toggle yes / no]', 'jetpack' ) }
```

Comment:
Since this is a toggle, do we need to add `[toggle yes / no]`?
```suggestion
					help={ __( 'Store in CRM', 'jetpack' ) }
```